### ISSUE TITLE
node-gyp rebuild

### ISSUE THREAD

**Issue by PearlQueenArasi** [stevetarver/excel-as-json#8]
When I am trying to install this package I am getting this error. My node version is v6.3.0 and npm version is 3.10.8 
File is attached for reference
[npm-debug.txt](https://github.com/stevetarver/excel-as-json/files/621482/npm-debug.txt)

**Comment by stevetarver** [comment]
This is a frustrating problem - dependencies not building on a specific platform. I don't have a windows machine to test your scenario with and the debug log (thanks for that). The only advice I can suggest is to resolve the libxmljs build problems in isolation and/or reach out to the libxmljs author for help.

```
911 error Failed at the libxmljs@0.18.0 install script 'node-gyp rebuild'.
911 error Make sure you have the latest version of node.js and npm installed.
911 error If you do, this is most likely a problem with the libxmljs package,
911 error not with npm itself.
```

good luck

**Comment by stevetarver** [comment]
I have just updated the 'excel` dependency and released version 1.0.4. You might try installing the latest version of this package and see if the problem magically resolved itself.

**Comment by stevetarver** [comment]
I found this in a previous issue in case it helps:

> If you search through that log for libxmljs, you don't find much more info than this. On my mac, node-gyp writes errors to stdout, so you could retry and look for those node-gyp errors for more details.
>
> It may be a node-gyp dependency problem - you should verify that all required dependencies are installed. Details are [here](https://github.com/nodejs/node-gyp#installation).
>
> libxmljs has had issues building on windows. [This open issue](https://github.com/libxmljs/libxmljs/issues/392) looks similar to your problem, was opened in v 0.17.0, so getting the `excel.js` author to update his version of libxmljs won't help.
